presto-docs/src/main/sphinx/release/release-0.285.rst

```diff
@@ -0,0 +1,75 @@
+=============
+Release 0.285
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix `min_by(x, y, n) / max_by(x, y, n)` functions to ensure `n` is unique.
+* Fix LIKE pattern to handle multi-byte characters. :pr:`21578`
+* Improve latency of HBO. :pr:`21297`
+* Improve HBO to track partial aggregation statistics. :pr:`21160`
+* Improve PushPartialAggregationThroughExchange to use partial aggregation statistics when available. This extends to multi-key aggregations.
+* Improve union node by removing the empty inputs.
+* Improve HBO latency by skipping retry if it times out. :pr:`21308`
+* Improve join-reordering to work with non-simple equi-join predicates.
+* Improve :doc:`REST API</rest>` documentation by providing links.
+* Improve :doc:`Resource Groups </admin/resource-groups>` documentation.
+* Add Presto to MySQL type mapping in :doc:`MySQL Connector</connector/mysql>` documentation.
+* Add :func:`array_least_frequent` function.
+* Add support for HANA connector. :pr:`21034`
+* Add task killer which is triggered when a worker is running out of memory and the garbage collector cannot reclaim sufficient memory. Two strategies are provided: full garbage collection, and frequent full garbage collection. :pr:`21254`
+* Add a session property ``history_input_table_statistics_matching_threshold`` to set the threshold for input statistics match in HBO. :pr:`21063`
+* Add a session property ``remove_redundant_cast_to_varchar_in_join`` (enabled by default) to remove redundant `cast to varchar` expressions in a join condition. :pr:`21050`
+* Add a session property ``enable_hbo_for_scaled_writer`` (disabled by default) to use HBO for scaled writers.
+* Add a session property ``use_partial_aggregation_history`` if partial aggregation statistics must be used to split aggregates into partial and final. :pr:`21160`
+* Add a config property ``task.memory-based-slowdown-threshold`` that provides a heap memory threshold below which new splits are not processed. :pr:`20946`
```

Comment:
This feature is experimental and dangerous, I do not recommend adding it to the release notes.